### PR TITLE
Adding configuration for id_minter service

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: terraform/queues.tf [hunk 1]
```diff
@@ -5,3 +5,11 @@ module "ingest_queue" {
   account_id  = "${data.aws_caller_identity.current.account_id}"
   topic_names = ["${aws_sns_topic.ingest_topic.name}"]
 }
+
+module "ingest_queue" {
```
Comment (alexwlchan) [inline terraform/queues.tf]: I think module names need to be unique – and this isn’t the ingest queue. 😉

File: terraform/queues.tf [hunk 2]
```diff
@@ -5,3 +5,11 @@ module "ingest_queue" {
...
+  source      = "./sqs"
+  queue_name  = "es_id_minter_queue"
```
Comment (alexwlchan) [inline terraform/queues.tf]: The `es_` prefix is for "Elasticsearch" – I think `id_minter_queue` would be enough?

File: terraform/sns.tf [hunk 3]
```diff
@@ -1,3 +1,7 @@
+resource "aws_sns_topic" "id_minter_topic" {
+  name = "es_id_minter"
```
Comment (alexwlchan) [inline terraform/sns.tf]: ditto above

File: terraform/iam.tf [hunk 4]
```diff
@@ -114,6 +119,27 @@ resource "aws_iam_role_policy" "ecs_transformer_task_sns" {
   name = "ecs_task_task_sns_policy"
   role = "${module.ecs_transformer_iam.task_role_name}"
 
+  policy = "${data.aws_iam_policy_document.publish_to_id_minter_sns.json}"
+}
+
+/** Allows publishing to the ingest topic. */
```
Comment (alexwlchan) [inline terraform/iam.tf]: Comment is wrong.

File: terraform/iam.tf [hunk 5]
```diff
@@ -114,6 +119,27 @@ resource "aws_iam_role_policy" "ecs_transformer_task_sns" {
...
+      "SNS:Publish",
+    ]
+
+    resources = [
+      "${aws_sns_topic.id_minter_topic.arn}",
+    ]
+  }
+}
+
+/** Allows the transformer app to publish to the ingest topic. */
```
Comment (alexwlchan) [inline terraform/iam.tf]: Ditto above.

**Comment by jamesgorrie** [comment]
What s a minter queue?

**Pull request merged**